### PR TITLE
CLUS-firepower-restapi111-md1

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* changed sentences
* removed capitls in headers
* removed estimated time
* added link for different lab

### CODE REVIEW

File: labs/firepower-restapi-111/1.md [hunk 1]
```diff
@@ -1,42 +1,42 @@
 # FMC - REST - API - 111
 
-## Lab Objective
+## Lab objective
```
Comment (paulzim) [inline labs/firepower-restapi-111/1.md]: Should just be "Objectives"

File: labs/firepower-restapi-111/1.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+
 ## Creating a TID Source
 
 ### To Create a TID Source using URL download
-1. Select API Operations on Source objects and choose the POST Operation.
-Based on the type of source and its contents you can modify the fields feedType, feedContent in the JSON sample below.
+1. Select API Operations on Source objects and choose the POST Operation. Based on the type of source and its contents, you can modify the fields `feedType` and `feedContent` in the JSON sample below.
 
-2. In the API CONSOLE post the JSON below to configure a STIX source which is located at: https://cisco.box.com/shared/static/hxlc1d1r8u3naaj876mqwucuqjuywx01.txt 
+2. In the API CONSOLE, post the JSON below to configure a [STIX source](https://cisco.box.com/shared/static/hxlc1d1r8u3naaj876mqwucuqjuywx01.txt).
```
Comment (paulzim) [inline labs/firepower-restapi-111/1.md:31]: Good!

**Review by paulzim — CHANGES_REQUESTED** [review]

File: labs/firepower-restapi-111/1.md [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
 
-Completion time: 30 minutes
 
 ## Prerequisites
-Basic understanding of REST principles
-Optional: Learning Lab "FMC - REST - API - 102" to get basic understanding of accessing FMC REST APIs using tokens.
+It's best if you already  have basic understanding of REST principles. Optionally, you can work through [Learning Lab "FMC - REST - API - 102](https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1) before this lab, in order to get basic understanding of accessing FMC REST APIs using tokens.
+
+
+## Learning objective
```
Comment (annegentle) [inline labs/firepower-restapi-111/1.md]: The heading, Learning objective, can be removed.

File: labs/firepower-restapi-111/1.md [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+After completing this module you will have a basic understanding of the available Firepower Management Center (FMC) TID APIs and their usage.
 
-## Learning Objective
-After completing this module you will have a
-basic understanding of the available FMC TID APIs and their usage.
 
 ## FMC REST API Explorer
-The API Explorer provides the list of APIs available on the Firepower Management Center (FMC), their details and the ability to make permissible Rest API Calls using a simplified API console.
+The API Explorer provides the list of APIs available on the FMC, their details, and the ability to make permissible Rest API Calls using a simplified API console.
+
```
Comment (annegentle) [inline labs/firepower-restapi-111/1.md]: Calls can be lowercase here.

File: labs/firepower-restapi-111/1.md [hunk 5]
```diff
@@ -106,15 +107,16 @@ Following are the key/value pairs:
 
 ![Figure: FMC REST API example setup](/posts/files/firepower-restapi-111/assets/images/Picture3.png)
 
-### To View Observables
 
-Observables represent stateful properties or measurable events pertinent to the operation of computers and networks E.g., IPs, Domains, URLs, File SHA’s etc.
+### To View observables
```
Comment (annegentle) [inline labs/firepower-restapi-111/1.md:107]: Maybe revise as "View observables?"

File: labs/firepower-restapi-111/1.md [hunk 6]
```diff
@@ -106,15 +107,16 @@ Following are the key/value pairs:
...
+
+Observables represent stateful properties or measurable events pertinent to the operation of computers and networks. For example, IPs, Domains, URLs, File SHA’s etc.
 
-* Select API Operations on observable objects and follow the steps described in the Indicators of Compromise section.
+Select API Operations on observable objects and follow the steps described in the Indicators of Compromise section.
 
 
-### To View Incidents
+### To View incidents
```
Comment (annegentle) [inline labs/firepower-restapi-111/1.md:114]: Could be revised to "View incidents."

File: labs/firepower-restapi-111/1.md [hunk 7]
```diff
@@ -1,42 +1,42 @@
...
-The API Explorer provides the list of APIs available on the Firepower Management Center (FMC), their details and the ability to make permissible Rest API Calls using a simplified API console.
+The API Explorer provides the list of APIs available on the FMC, their details, and the ability to make permissible Rest API Calls using a simplified API console.
+
 
 ### Using API Explorer:
-1. Navigate to the URL: https://management_center_IP_or_name/api/api-explorer Or try the DevNet live sandbox Or try the DevNet live sandbox https://cisco-tid.cisco.com/api/api-explorer/
+1. Navigate to the https://management_center_IP_or_name/api/api-explorer URL. Or try the [DevNet Live Sandbox](https://cisco-tid.cisco.com/api/api-explorer/).
 
-2. Log in with your username and password and click "Sign In"
+2. Log in with your username and password and click "Sign In".
```
Comment (annegentle) [inline labs/firepower-restapi-111/1.md:19]: We should probably say more about how people get the credentials for the DevNet Sandbox... as I understand it, they don't want to publish the actual credentials here though... let's find someone with a definitive answer.

**Review by annegentle — CHANGES_REQUESTED** [review]

**Comment by ReneeBornstein** [comment]
This one, too. Maybe you did something with the reviews that made it not allow me to do any changes?

OMG, another weird error. I opened the GitHub doc link you sent, clicked on Files changed, and copied that to another window, so I could edit it. Usually I then just click on the pencil icon to edit the file. But this time I got this error:

[/var/folders/9x/dk6rpbtd6tnc0tqvtk73lz100000gn/T/com.microsoft.Outlook/WebArchiveCopyPasteTempFiles/cidimage001.png@01D40BD1.FE73A850]

I’ve never had that happen before. I am logged in, so I guess somehow my push access (whatever that is) suddenly isn’t working?

**Review by annegentle — APPROVED** [review]

**Pull request merged**